### DISCUSSION TITLE
开启国际化插件为啥一定需要依赖antd？

### DISCUSSION THREAD

**Discussion by ratelgogo** [umijs/umi#12453] [General]
一般移动端的项目都不会使用antd包，但是有国际化的需求，使用@umijs/max的国际化插件不安装antd时会构建失败，是bug还是feature？

**Comment by fz6m** [comment]
国际化插件和 antd 不是强绑定的，你是开启了 `locale.antd: true` 选项了吗。

如还有问题，需要给一个最小复现。

**Reply by ratelgogo** [reply]
可以了，locale.antd 默认是true的吧，显式设置false就ok了，感谢🙏
